lib/pact_broker/api/decorators/webhook_decorator.rb

```diff
@@ -16,7 +16,7 @@ class WebhookEventDecorator < BaseDecorator
           property :name
         end
 
-        property :uuid
+        property :uuid, writeable: false
```

Comment:
Dammit. Forgot to push this commit before making PR. Wish I knew how to get rid of these commits when this happens.